tests/generate_cases/simple_types/integer/multipleOf.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "integer",
```

Comment:
Two spaces or four? I prefer four for JSON.

Reply:
Once this is merged in I can open a separate PR to standardize the JSON formatting

Reply:
I am indifferent but since JQ uses two I prefer two.